.github/workflows/ci.yml

```diff
@@ -0,0 +1,137 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+      - release/**
+
+  pull_request:
+
+jobs:
+  lints:
+    name: Lints
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
+
+      - name: Install python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
```

Comment:
do we want this explicitly? python 3.8 is the default

Reply:
Yes, we want to explicitly control which version of python we use IMO, even if it happens to match the system default one currently